packages/rest-api-client/src/client/types/space/index.ts

```diff
@@ -0,0 +1,21 @@
+import type { App } from "../app";
+
+type AttachedApp = Omit<App, "spaceId">;
+
+export type Space = {
+  id: string;
+  name: string;
+  defaultThread: string;
+  isPrivate: boolean;
+  creator: { code: string; name: string };
+  modifier: { code: string; name: string };
+  memberCount: string;
+  coverType: string;
```

Comment:
Please narrow the type based on the document.

```suggestion
  coverType: "BLOB" | "PRESET";
```